services/cryptoService.ts

```diff
@@ -0,0 +1,30 @@
+import {CryptoCurrency} from '../utils/types/crypto';
+import axios from 'axios';
+
+const fetchCryptoData = async (currency: string): Promise<CryptoCurrency> => {
+  try {
+    const API_BASE_URL = `https://data.messari.io/api/v1/assets/${currency}/metrics`;
```

Comment:
https://data.messari.io/api/v1/assets 
should be stored in a .env file

https://yarnpkg.com/package/react-native-dotenv